homeassistant/components/calendar/caldav.py

```diff
@@ -138,9 +144,11 @@ def update(self):
         # won't return events that have already started
         results = self.calendar.date_search(
             dt.start_of_local_day(),
-            dt.start_of_local_day() + timedelta(days=1)
+            dt.start_of_local_day() + timedelta(days=3)
```

Comment:
I guess you're increasing the range here to also capture events which are far in the future, in case we set a very long offset time? Though three days is probably safe, it would be less error-prone to pass the offset time into the `WebDavCalendarData` object and then use `dt.start_of_local_day() + timedelta(days=1) + self._offset_time` here.